resources/translations/en.edn

```diff
@@ -170,15 +170,15 @@
          :apply "Apply"
          :apply-for-bundle "Apply for %1 resources as a bundle "
          :header "%1 resources in the cart"
-         :intro [:div.intro [:p "Add resources to the cart to apply for them. If several resources can be applied to with the same application, they will be automatically grouped into a bundle."]]
+         :intro [:div.intro [:p "Apply for access by adding resources to the cart. When applying for multiple rights at the same time, resources using the same application process are bundled together as one application."]]
          :remove "Remove"}
   :catalogue {:catalogue "Catalogue"
               :header "Resource"
-              :intro [:p "The catalogue contains all the resources that you can apply for in this system."]
+              :intro [:p "Access can be applied for the following resources."]
```

Comment:
this seems ungrammatical, maybe "Access to the following resources can be applied for"

Reply:
I think the problem is that "apply" needs the dependent preposition ("apply for"), so grammatically it should be "Access can be applied for for the following resources", but having to repeat "for" doesn't sound that good.

Reply:
We had a long discussion about the "for" problem last week. "Access to the following resources can be applied for" would work.

Reply:
Done